src/scielo/bin/xml/prodtools/utils/xml_utils.py

```diff
@@ -213,11 +213,7 @@ def format(self, pretty_print=False, dtd_location_type=None):
                 xml_declaration=self.xml_declaration,
                 pretty_print=pretty_print, doctype=doctype
                 ).decode("utf-8")
-        return "\n".join([
-                self.xml_declaration,
-                self.doctype,
-                self.content,
-            ])
+        return self.original
```

Comment:
O método passa a ter um comportamento inconsistente por ignorar o argumento `pretty_print` no caso onde `self.xml is None`.

Reply:
O melhor seria retornar `None` aqui e não usar o método `.format()`? Inconsistente realmente já estava.

Reply:
isso terá um efeito colateral no resto do programa.

Reply:
Qual o significado associado ao fato de `self.xml` ser `None`? O _setter_ de _content_ me leva a pensar que seria por erro no _parsing_ do arquivo/string de bytes apontado como XML. Nesse caso `self.original` seria um XML quebrado, é isso?

Reply:
> isso terá um efeito colateral no resto do programa. Então, se o retorno for `None`, ok, mas corrigir nos pontos onde `format` é chamado.

@robertatakenaka você acha que seria ruim alterar o `.write()` para gravar o conteúdo original como veio ao invés de gravar o retorno do `.format()`?

> Qual o significado associado ao fato de `self.xml` ser `None`? O _setter_ de _content_ me leva a pensar que seria por erro no _parsing_ do arquivo/string de bytes apontado como XML.

@gustavofonseca, pela minha análise superficial do código, eu diria que você está certo.

> Nesse caso `self.original` seria um XML quebrado, é isso?

Isso.

Reply:
Sobre efeitos colaterais, acho que qualquer alteração no método `.format()` ou `.write()` pode causar efeitos colaterais no resto do programa, visto que a classe `SuitableXML` é usada em vários pontos, inclusive em uma herança.

Reply:
Me parece que retornar `None` é mais adequado, mas como a @robertatakenaka bem disse será necessário alterar os pontos de invocação -- eu identifiquei que um deles é o método `write`, mas não fiz uma busca ampla pela base de códigos.

Reply:
Eu fiz essa busca e os únicos lugares são mesmo no método `write` e nos testes unitários (deu trabalho buscar por um método que tem exatamente o mesmo nome de outro tão comumente usado).
Talvez a análise necessária aqui seja quais as saídas esperadas por essa classe. Roberta pontuou que é importante que essa classe produza um XML, mesmo que inválido, pois seria possível identificar no XML resultado onde está o erro. Não podemos produzir um XML sem conteúdo. Resta saber se é melhor gravar o XML original, sem alteração alguma, ou parcialmente corrigido e verificar os impactos no restante da aplicação.

Reply:
@patymori acho que este PR pode ser fechado, sem fazer merge, eu vou trabalhar neste issue.